frontend/src/components/task/task.js

```diff
@@ -693,9 +693,6 @@ class Task extends Component {
       this.props.history.push('/404')
       return null
     }
-    if (!task.completed) {
-      return (<div />)
-    }
```

Comment:
I actually wanted to render a empty screen when the task didn't get rendered. But now we have a loader that's completely unnecessary so removed it.